uploadNeuroDB/bin/mincpik

```diff
@@ -1,4 +1,156 @@
 #! /usr/bin/env perl
+
+=pod
+
+=head1 NAME
+
+mincpik - generate an image from a set of MINC volumes
+
+=head1 SYNOPSIS
+
+mincpik [options] minc_file [outfile]
```

Comment:
Maybe mention the the available [options] are listed in the description?

Reply:
I suggest to leave it like this (that's the way the synopsis is described in a Unix manual page)